src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -212,9 +212,7 @@ const AppWrapper = (props: { children: ReactNode }) => {
       // ]);
     };
     const removeConnectionsPendingDeletion = async () => {
```

Comment:
This function should not be in the UI AT ALL. `removeConnectionsPendingDeletion` has to be set up in the core and in the core only.